karma/makeconf.js

```diff
@@ -30,7 +30,7 @@ function makeConf(defaultFile, browserNoActivityTimeout, requires) {
         throw new Error('Unknown browser');
       }
     } else if (process.platform === 'darwin') {
-      browsers = ['ChromeWebRTC', 'FirefoxWebRTC', 'Safari', 'Electron'];
+      browsers = ['ChromeWebRTC',  'Electron', 'FirefoxWebRTC', 'Safari' ];
```

Comment:
@makarandp0 Nitpicks:
* Can you remove the extra space before `'Electron'`?
* Can you remove the extra space after `'Safari'`?

Reply:
fixed now (along with with some updates for customizing electron)